docs/getting-started.asciidoc

```diff
@@ -0,0 +1,224 @@
+[[getting-started]]
+== Getting Started with Kibana
+
+Now that you have Kibana <<setup,installed>>, you can step through this tutorial to get fast hands-on experience with 
+key Kibana functionality. By the end of this tutorial, you will have:
+
+* Loaded a sample data set into your Elasticsearch installation
+* Defined at least one index pattern
+* Used the <<discover, Discover>> functionality to explore your data
+* Set up some <<visualize,_visualizations_>> to graphically represent your data
+* Assembled visualizations into a <<dashboard,Dashboard>>
+
+The material in this section assumes you have a working Kibana install connected to a working Elasticsearch install.
+
+[float]
+[[tutorial-load-dataset]]
+=== Before You Start: Loading Sample Data
+
+The tutorials in this section rely on the following data sets:
+
+* The complete works of William Shakespeare, suitably parsed into fields. Download this data set by clicking here: 
+https://www.elastic.co/guide/en/kibana/3.0/snippets/shakespeare.json[shakespeare.json].
+* A set of fictitious accounts with randomly generated data. Download this data set by clicking here: 
+https://github.com/bly2k/files/blob/master/accounts.zip?raw=true[accounts.json]
+
+The Shakespeare data set is organized in the following schema:
+
+[source,json]
+{
+    "line_id": INT,
+    "play_name": "String",
+    "speech_number": INT,
+    "line_number": "String",
+    "speaker": "String",
+    "text_entry": "String",
+}
+
+The accounts data set is organized in the following schema:
+
+[source,json]
+{
+    "account_number": INT,
+    "balance": INT,
+    "firstname": "String",
+    "lastname": "String",
+    "age": INT,
+    "gender": "M or F",
+    "address": "String",
+    "employer": "String",
+    "email": "String",
+    "city": "String",
+    "state": "String"
+}
+
+After downloading the data sets, load them into Elasticsearch with the following commands:
+
```

Comment:
We should really have the user import a mapping. Mappings are so important in elasticsearch that learning about them later on is sort of harmful. See this step in the Kibana 3 tutorial: https://www.elastic.co/guide/en/kibana/3.0/import-some-data.html